### DISCUSSION TITLE
Need help keeping 1080p constant for Korean channels

### DISCUSSION THREAD

**Discussion by erichyunjunsong** [iptv-org/community#64] [Q&A] [answered]
I've been trying to watch some Korean channels, namely JTBC (1080p) and TVN (1080p), which initially load in at 1080p then drop down in quality significantly. Is there anything I can do on my end to keep that quality constant? 

I am trying to watch the channels from Australia, with a 100mb down/20mb up internet connection, using no VPNs.

**Comment by BellezaEmporium** [comment, marked as answer]
These ones have got a fairly poor bandwidth / stream management. They're from the SeeZN services.

The only way of getting them correctly is by sticking to the 404p stream. Lacks on quality, but at least it doesn't cut.

I have 800mb/600mb down/up and it's difficult for me too.

**Reply by erichyunjunsong** [reply]
Thanks for the response! Guess I'll have to try find a separate IPTV service that I could buy that has those channels
